### DISCUSSION TITLE
Are mermaid.js markdown strings supported?

### DISCUSSION THREAD

**Discussion by chrisdburr** [squidfunk/mkdocs-material#5859] [Q&A] [answered]
Is the Markdown strings functionality of Mermaid.js supported in Material for MK Docs or is it still a new feature waiting to be implemented?

Mermaid.js Markdown Strings documentation: https://mermaid.js.org/syntax/flowchart.html#markdown-strings

Not sure if this is the same issue as mentioned here: https://github.com/squidfunk/mkdocs-material/pull/5771

In case it helps, here is a simple flowchart diagram that renders correctly using [Mermaid's live editor](https://mermaid.live/), but does not display the bold, italics, or line breaks correctly in Material.

```mermaid
graph TD

A["`A sentence with **bold** text
split across two lines.`"] --> B;
B["`Some *italics*.`"];
```

**Comment by chrisdburr** [comment]
Looks like it doesn't display as intended in GitHub discussions either 👆

Here's the raw text in case it helps:

```
graph TD

A["`A sentence with **bold** text
split across two lines.`"] --> B;
B["`Some *italics*.`"];
```

**Reply by squidfunk** [reply, marked as answer]
If it was added in Mermaid.js 10+, it's blocked by #5758

**Reply by chrisdburr** [reply]
Thanks. It does indeed seem like it was added in version 10+ 

https://www.mermaidchart.com/blog/posts/automatic-text-wrapping-in-flowcharts-is-here

**Reply by squidfunk** [reply]
Jup, if somebody wants to tackle it, we're happy to merge a PR. Other than that, we'll work on it once the dust settles on 9.2.

**Reply by galindro** [reply]
I am looking forward this feature as well. Hasn't it been implemented yet? If not, is there any workaround to make it work?

**Reply by squidfunk** [reply]
We're on the latest version of Mermaid.js, but I'm not sure what needs to be done to make it work.

**Reply by galindro** [reply]
nervermind... just discovered that I was using an old material for mkdocs version. It is working like a charm. ty :)
